gdal/ogr/ogrsf_frmts/segy/ogrsegylayer.cpp

```diff
@@ -492,6 +526,8 @@ OGRFeature *OGRSEGYLayer::GetNextRawFeature()
     const double dfGroupX = nGroupX * dfHorizontalScale;
     const double dfGroupY = nGroupY * dfHorizontalScale;
 
+    dfHorizontalScale = nHorizontalScalar;
```

Comment:
is this needed ?

Reply:
No. You are right, it's a mistake.

Reply:
Can you push an additional commit to remove this ? thanks